src/Farmer/Builders/Builders.ContainerService.fs

```diff
@@ -17,7 +17,19 @@ type AgentPoolConfig =
       VmSize : VMSize
       VirtualNetworkName : ResourceName option
       SubnetName : ResourceName option }
-
+    static member Default = {
+            Name = ResourceName.Empty
```

Comment:
Unless GH is showing wrongly here, this is double indented (8 instead of 4 chars).